view-admin/src/components/common/JudgementModal/JudgementModal.tsx

```diff
@@ -64,6 +71,26 @@ const Modal = ({
     fetchBingoNumbers();
   }, [bingoNumbers]);
 
+  function BingoJudgement() {
+    if (isIncluded) {
+      return (
+        <>
+          <div className={styles.jugementResults}>
+            <p>BINGO</p>
+          </div>
+        </>
+      );
+    } else {
+      return (
+        <>
+          <div className={styles.jugementResults}>
+            <p>NotYet!</p>
+          </div>
+        </>
+      );
+    }
+  }
+
```

Comment:
if文を三項演算子で処理すれば1msくらいは早くなるかも？
```suggestion
  const BingoJudgement = () => (
    <div className={styles.jugementResults}>
      <p>{isIncluded ? "BINGO" : "NotYet!"}</p>
    </div>
  );
  
```